eclair-core/src/main/scala/fr/acinq/eclair/channel/Commitments.scala

```diff
@@ -41,13 +41,13 @@ case class WaitingForRevocation(nextRemoteCommit: RemoteCommit, sent: CommitSig,
 // @formatter:on
 
 /**
- * about remoteNextCommitInfo:
- * we either:
- * - have built and signed their next commit tx with their next revocation hash which can now be discarded
- * - have their next per-commitment point
- * So, when we've signed and sent a commit message and are waiting for their revocation message,
- * theirNextCommitInfo is their next commit tx. The rest of the time, it is their next per-commitment point
- */
+  * about remoteNextCommitInfo:
```

Comment:
For some reason latest version of Intellij formats block comment differently. Any chance you can update your settings? Otherwise we'll have constant back and forth.

Reply:
I've updated to `2019.2` and let it auto-format comments.